msg/templates/uorb_microcdr/microRTPS_client.cpp.em

```diff
@@ -149,7 +168,7 @@ static int launch_send_thread(pthread_t &sender_thread)
 {
     pthread_attr_t sender_thread_attr;
     pthread_attr_init(&sender_thread_attr);
-    pthread_attr_setstacksize(&sender_thread_attr, PX4_STACK_ADJUSTED(4000));
+    pthread_attr_setstacksize(&sender_thread_attr, PX4_STACK_ADJUSTED(4096));
```

Comment:
How much does it need now? (after the dynamic allocation)

Reply:
I am actually making it consistent with the main task. But actually 4000 still works.

Reply:
Consistency does not matter for that, make it as small as required (actual use + 350 bytes). Otherwise it's just wasting RAM.
The same for https://github.com/PX4/Firmware/blob/master/src/modules/micrortps_bridge/micrortps_client/microRTPS_client_main.cpp#L199.

Reply:
Thanks for the feedback. Solved in https://github.com/PX4/Firmware/pull/15354/commits/730b1fe13494b411d5fb55cab285cb91d46cc71f.